src/annotate.ts

```diff
@@ -1,9 +1,20 @@
 import * as _request from 'request-promise';
+import * as Ajv from 'ajv';
+import * as fs from 'fs';
 
 //-----------------------
 
 export default async function annotate(opts: { payload: { field: string, term: string } }) {
 
+  const schema: any = JSON.parse(fs.readFileSync('./schemas/is-valid.json', 'utf8'));
```

Comment:
I know is a WIP but remember you should move the loading of the schema outside of the annotate function, mainly when when the loading function `readFileSync` is blocking the thread.